incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -530,6 +529,18 @@ namespace if this annotation already exists. The two are mutually exclusive.
 We are considering replacing this with the standard
 `app.kubernetes.io/managed-by` label in the future.
 
+#### hnc.x-k8s.io/excluded-namespace (label on namespaces)
+
+This label is supposed to be added to the excluded namespaces such as
+`kube-system`, `kube-public`, and `hnc-system` so that when HNC validating
+webhook services are down, operations in these excluded namespaces won't be
+affected.
```

Comment:
Suggestion:

This label should be added to namespaces such as `kube-system` and `kube-public` so that HNC's validating webhook cannot accidentally prevent operations in these namespaces and block critical cluster operations.